environment.yml

```
@@ -44,7 +44,7 @@ dependencies:
     - torchtext>=0.8.*
```

Should we update the min version of torchtext as well?
```suggestion
    - torchtext>=0.9.*
```

Same applies to:
https://github.com/PyTorchLightning/pytorch-lightning/blob/aea0a9b218aa9fe596684953eb8134154cf6d334/requirements/extra.txt#L5

(Its version matrix is here: https://github.com/pytorch/text#installation)

---

Yes

---

Thanks, @akihironitta - I missed this, it makes sense. :)